mmaction/core/evaluation/ava_utils.py

```diff
@@ -0,0 +1,267 @@
+import csv
+import heapq
+import logging
+import time
+from collections import defaultdict
+
+import numpy as np
+
+from .ava_evaluation import object_detection_evaluation as det_eval
+from .ava_evaluation import standard_fields
+from .recall import eval_recalls
+
+
+def det2csv(dataset, results):
+    csv_results = []
+    for idx in range(len(dataset)):
```

Comment:
After double check, `range(len(x))` should be used here, since the meaning of ` for idx in range(len(dataset))` and `for idx, _ in enumerate(dataset)` is different. If the later one is used. `__getitem__` of dataset will be called, which will be extremely slow.

Reply:
Agree